src/App.js

```diff
@@ -36,6 +38,7 @@ class App extends React.Component {
           <Header user={this.state.user} onLogout={this.logoutHandler} />
           <Switch>
             <Route exact path="/">
+            <BestBooks />
               {/* TODO: if the user is logged in, render the `BestBooks` component, if they are not, render the `Login` component */}
             </Route>
             {/* TODO: add a route with a path of '/profile' that renders a `Profile` component */}
```

Comment:
Nice job using router for `BestBooks` and home(`/`). Use React Router here to also add ability for user to navigate between `Profile` and `Home` "pages".